src/convertToObject.js

```diff
@@ -6,7 +6,29 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // First - clear string from other symbols
+  const tempString = sourceString.trim().replace(/;+\s*$/, '');
+  const cssList = tempString.split(';');
+  const cssObject = {};
+
+  cssList.forEach((style) => {
+    let [key, value] = style.split(':');
+
+    if (key && value) {
+      key = key.trim();
+      value = value.trim();
+
+      if (key && value) {
+        cssObject[key] = value;
+      }
+    }
+
+    return cssObject;
```

Comment:
The 'forEach' loop is correctly used to iterate over the style declarations. However, the return statement inside the 'forEach' callback is unnecessary because 'forEach' does not use the returned value from the callback. You can remove the return statement to avoid confusion.